pipelines/batch/src/main/java/org/openmrs/analytics/JdbcFetchHapi.java

```diff
@@ -96,7 +99,7 @@ public void setParameters(List<String> element, PreparedStatement preparedStatem
 					            preparedStatement.setInt(2, Integer.valueOf(element.get(1)));
 					            preparedStatement.setInt(3, Integer.valueOf(element.get(2)));
 				            }
-			            }).withOutputParallelization(true)
+			            }).withOutputParallelization(false)
```

Comment:
Please add a comment why we are disabling this.

As a side note, I just remembered that long time ago we had a [discussion](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/72#discussion_r557062588) about this method with Allan when he first implemented the JDBC support for OpenMRS. Back then it was not clear why this needs to be disabled but now I am thinking that maybe we should do it in that case too (this is just FYI).

Reply:
Thanks for letting me know; I've added the comment on the reasoning behind disabling this parameter.